linter-rules.json

```diff
@@ -0,0 +1,55 @@
+{
+    "version": 1,
+    "lineLength": 9999,
+    "maximumBlankLines": 2,
+    "tabWidth": 4,
+    "indentation": {
+        "spaces": 4
+    },
+    "respectsExistingLineBreaks": true,
+    "lineBreakBeforeControlFlowKeywords": false,
+    "lineBreakBeforeEachArgument": false,
+    "lineBreakBeforeEachGenericRequirement": false,
+    "indentConditionalCompilationBlocks": false,
+    "prioritizeKeepingFunctionOutputTogether": true,
+    "lineBreakAroundMultilineExpressionChainComponents": true,
+    "spacesAroundRangeFormationOperators": true,
+    "indentSwitchCaseLabels": false,
+    "rules": {
+        "AllPublicDeclarationsHaveDocumentation": false,
+        "AlwaysUseLowerCamelCase": false,
+        "AmbiguousTrailingClosureOverload": true,
+        "BeginDocumentationCommentWithOneLineSummary": false,
+        "DoNotUseSemicolons": true,
+        "DontRepeatTypeInStaticProperties": true,
+        "FileScopedDeclarationPrivacy": true,
+        "FullyIndirectEnum": true,
+        "GroupNumericLiterals": true,
```

Comment:
This one could be controversial. (It basically formats numeric literals like `10_000_000`. I turned it on because it's mostly automatically and it does look nice.